css/src/components/table.css

```diff
@@ -0,0 +1,168 @@
+.yoast table {
```

Comment:
This will target _any_ table within a Yoast page, which looks not so safe to me. I think we should provide a `get_table_classes()` method in all our classes that extend `WP_List_Table`. That will allow us to add our own classes to our `<table>` elements.